segments-guide.md

```diff
@@ -0,0 +1,327 @@
+
+## Retrieving Segments
+
+When requesting a list of segments, you can use multiple URL query parameter filters. You can filter by `name`, `tagNames`, segment id (`segmentFilter`), and `rsids`. For example, to retrieve all segments tagged as part of the *SpringPromotion* campaign, you can add the URL parameter `tagNames=SpringPromotion`. The `tagNames`, `segmentFIlter`, and `rsids` filters accept comma-delimited lists. These lists should be short, with no more than 100 items.
+
+### Example cURL Request
+
+The following example requests all segments tagged as part of the *SpringPromotion* campaign with `locale` value `en_US`, on page 0 with page size 10:
+
+```
+
+curl -X GET "https://analytics.adobe.io/api/cellina/segments?locale=en_US&tagNames=SpringPromotion&limit=10&page=0" -H "x-api-key: {OAUTHTOKEN}" -H "x-proxy-global-company-id: {COMPANYID}" -H "Authorization: Bearer {ACCESSTOKEN}" -H "Accept: application/json" -H "Content-Type: application/json"
+
+```
+
+### Example Response
+
+The following response shows six segments tagged as part of the *SpringPromotion* campaign returned on one page:
+
+```
+
+{
+   "content":[
+      {
+         "id":"s300000022_591a105ce4b0fc8647cec9ae",
+         "name":"non-oberon segment",
+         "description":"non-oberon segment",
+         "rsid":"obue.analytics.spa",
+         "owner":{
+            "id":596983
+         }
+      },
+      {
+         "id":"s300000022_5924b570e4b0b06b9cc6e74b",
+         "name":"LONDON",
+         "description":"",
+         "rsid":"obue.appservicesprod",
+         "owner":{
+            "id":596983
+         }
+      },
+      {
+         "id":"s300000022_5b9fe2a19678905e58c6f2b6",
+         "name":"test aam",
+         "description":"test aam",
+         "rsid":"sistr2.beta",
+         "owner":{
+            "id":596983
+         }
+      },
+      {
+         "id":"s300000022_5b9fee889678905e58c6f2c3",
+         "name":"test aam 2",
+         "description":"test aam2",
+         "rsid":"sistr2.beta",
+         "owner":{
+            "id":596983
+         }
+      },
+      {
+         "id":"s300000022_5b9fef419678905e58c6f2c5",
+         "name":"test aam 3",
+         "description":"test aam 3",
+         "rsid":"sistr2.beta",
+         "owner":{
+            "id":596983
+         }
+      },
+      {
+         "id":"s300000022_5bb7c94e80f0073611afb35c",
+         "name":"test create",
+         "description":"",
+         "rsid":"sistr2.beta",
+         "owner":{
+            "id":596983
+         }
+      }
+   ],
+   "totalElements":6,
+   "firstPage":true,
+   "numberOfElements":6,
+   "totalPages":1,
+   "lastPage":true,
+   "sort":null,
+   "size":10,
+   "number":0
+}
+
+```
+
+## Retrieving a Single Segment
+
+You can retrieve segments individually if the segment `id` is known. You can get the segment `id` from  report descriptions or other Analytics screens.
```

Comment:
While you can scrape a segment ID from the segment management page in the UI it isn't something that I would want to document. We may just want to tell them that you can get segment ids report descriptions or calling the get multiple segment endpoint.